docs/source/docs/calibration/calibration.rst

```diff
@@ -108,6 +105,13 @@ Calibration images can also be extracted from the downloaded JSON file using `th
    :width: 600
    :alt: Captured calibration images
 
+Calibrating using calibdb
```

Comment:
These are the boards they sell right? Not to be confused with the calibration website?